pom.xml

```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <!-- Minimal version required by Pipeline dependencies in 2.14.0 -->
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.159</jenkins.version>
```

Comment:
Can you explain why exactly this version? Why not just 2.60.3? This is currently the minimum required version for JCasC - see [pom.xml](https://github.com/jenkinsci/configuration-as-code-plugin/blob/configuration-as-code-1.35/pom.xml#L25).

Reply:
I cannot recommend using 2.60.3 I'd the LTS 2.164.3
Due to security issues with previous Jenkins core.

Reply:
I don't share this suggestion to force Jenkins upgrades by increasing the minimum required Jenkins version in a plugin. Upgrading Jenkins should be a different concern. Otherwise every plugin would have to change this field on every new Jenkins LTS release (with security fixes) if this suggestion is strictly followed.

If I am missing something, please let me know.

Reply:
https://jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/